ui/imports/shared/stores/BIP39_en.qml

```diff
@@ -12,8 +12,10 @@ ListModel {
             if (xhr.readyState === XMLHttpRequest.DONE) {
                 root.words = xhr.responseText.split('\n');
                 for (var i = 0; i < root.words.length; i++) {
-                    if (root.words[i] !== "") {
-                        insert(count, {"seedWord": root.words[i]});
+                    let word = root.words[i]
```

Comment:
Fixing the windows issue where the speed phrase could not be validated due to extra return carriage.